docs/Windows.md

```diff
@@ -1,216 +1,32 @@
 # Getting Started with Swift on Windows
 
-## clang (cross-compiling)
+This document summarizes the state of Swift on Windows. One can build and run
+Swift natively, or through the Windows Subsystem for Linux.
```

Comment:
I think that the fact that this document summarizes the state of Swift on Windows is implicit.